tests/nightly/test_large_array.py

```diff
@@ -952,11 +1001,11 @@ def test_reshape_like():
 
 
 def test_flatten():
-    a = create_2d_tensor(rows=LARGE_X, columns=SMALL_Y).reshape((LARGE_X//2, 2, SMALL_Y))
-    b = nd.flatten(a)
-    assert b[-1][-1] == (LARGE_X-1)
```

Comment:
why are we removing these asserts?

Reply:
It is related to the different precision of `int64` and `float32`. This function `create_2d_tensor` will generate a NDArray with the elements varies from 0 to `LARGE_X-1`. For `float32`, it will lose some precision when `LARGE_X` is too large, that is `LARGE_X - 1` or `LARGE_X - 2` can not represent the accurate value. This is different with `int64`.  So I think these asserts can be removed.